core/src/main/java/com/linecorp/armeria/client/endpoint/DynamicEndpointGroupBuilder.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2017 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.client.endpoint;
+
+import com.linecorp.armeria.common.annotation.Nullable;
+import com.linecorp.armeria.common.annotation.UnstableApi;
+
+/**
+ * Builds a new {@link DynamicEndpointGroup}.
+ */
+@UnstableApi
+public final class DynamicEndpointGroupBuilder
+        extends AbstractDynamicEndpointGroupBuilder {
+
+    @Nullable
+    private EndpointSelectionStrategy selectionStrategy;
+
+    DynamicEndpointGroupBuilder() {}
+
+    /**
+     * Sets the {@link EndpointSelectionStrategy} of the {@link DynamicEndpointGroup}.
+     * If unspecified, {@link EndpointSelectionStrategy#weightedRoundRobin()} is used.
+     */
+    public DynamicEndpointGroupBuilder selectionStrategy(EndpointSelectionStrategy selectionStrategy) {
+        this.selectionStrategy = selectionStrategy;
```

Comment:
Should check for nullability since the parameter `selectionStrategy` isn't marked `@Nullable`.
Related to this, what do you think of also giving `selectionStrategy` a default value `weightedRoundRobin ` instead of `null`?

Reply:
Done